packages/apollo-server-integration-testsuite/src/ApolloServer.ts

```diff
@@ -59,9 +59,12 @@ import FakeTimers from '@sinonjs/fake-timers';
 import type { AddressInfo } from 'net';
 import request, { Response } from 'supertest';
 import {
-  type KeyValueCache,
   InMemoryLRUCache,
+  type KeyValueCache,
 } from '@apollo/utils.keyvaluecache';
+// this imports from dist because we're performing `instanceof` checks and they
```

Comment:
this feels scary, since this is a test can we do some sort of `.constructor` string test or something?

Reply:
[5c299de](https://github.com/apollographql/apollo-server/pull/6536/commits/5c299de9d30d9edff9a4680aabb45377e61d5f97)